pilot/pkg/networking/core/v1alpha3/accesslog.go

```diff
@@ -174,28 +174,20 @@ func buildFileAccessLogHelper(mesh *meshconfig.MeshConfig) *accesslog.AccessLog
 			},
 		}
 	case meshconfig.MeshConfig_JSON:
-		var jsonLog *structpb.Struct
+		jsonLogValue := structpb.Value{}
```

Comment:
nit: why change this from struct to value? Seems like we later skip anyways if its not a struct but then don't get a useful error message potentionally?

Reply:
yeah good point, 8e163b2 changed it to just use one `structpb.Struct` to parse it